src/smclarify/bias/metrics/common.py

```diff
@@ -175,6 +175,47 @@ def ensure_series_data_type(series: pd.Series, values: Optional[List[Any]] = Non
     raise ValueError("Data series is invalid or can't be classified as neither categorical nor continous.")
 
 
+def ensure_label_value_data_types_for_categorical(series: pd.Series, positive_label_values: List[Union[str, int, float]]) -> List:
```

Comment:
naming of the function? maybe say convert or something?

Reply:
improved